dts/bindings/sensor/silabs,si7006.yaml

```diff
@@ -1,8 +1,14 @@
-# Copyright (c) 2019, Electronut Labs
```

Comment:
Please don't remove old copyright entries.

Reply:
It's a complete re-write of the file.

Reply:
Agree with @fkokosinski

Reply:
Added back.